object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,17 @@
+/* exported omit */
+
+// create a storage for the output that holds a copy of the source object.
+// look at all the keys.
+// if the key exists as a trait in the source copy,
+// then delete that key from the source copy.
+// return the output.
+
+function omit(source, keys) {
+  var output = { ...source };
```

Comment:
If you can explain what this is doing to me, I'll let you keep it 🤓

Reply:
i'll come find you during one of these breaks 😅